constellation-app/constellation
# Bug/fix help images not loading on paths with spaces

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Small fix when handling request to replace encoded spaces in file paths with actual spaces. Also renamed all `.PNG` files to `.png`. Which was causing images to not load on linux help pages.

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Images will now display, even if there is a space in the filepath.

### Possible Drawbacks


None.

### Verification Process



1. Build consty into an application, if you are not using a consty application already.
2. Move the whole consty application into a folder that has spaces in its name. If required, create on yourself.
3. Start consty
4. Open any help page
5. Observe images load correctly

To verify issue exists on older versions, repeat the steps above with an older version and observe images not loading on any help page.

### Applicable Issues


https://github.com/constellation-app/constellation/issues/2135
